standardtracer/tracer.go

```diff
@@ -22,10 +73,21 @@ func New(recorder SpanRecorder) opentracing.Tracer {
 	return rval
 }
 
+// New creates and returns a standard Tracer which defers completed Spans to
+// `recorder`.
+// Spans created by this Tracer support the ext.SamplingPriority tag: Calling
+// SetTag(ext.SamplingPriority, nil) causes the Span to be Sampled from that
+// point on.
+func New(recorder SpanRecorder) opentracing.Tracer {
+	opts := DefaultOptions()
+	opts.Recorder(recorder)
+	return NewWithOptions(opts)
+}
+
 // Implements the `Tracer` interface.
 type tracerImpl struct {
+	*Options
```

Comment:
Ok, enough flip-flopping: Introducing the interface makes the *Option the best choice after all.